authorizer/notebook_test.go

```diff
@@ -61,7 +61,7 @@ func Test_GetNotebook(t *testing.T) {
 				GetNotebook(gomock.Any(), *nbID).
 				Return(newTestNotebook(*orgID1), nil)
 
-			perm := newTestPermission(influxdb.ReadAction, tt.permissionOrg)
+			perm := newTestNotebooksPermission(influxdb.ReadAction, tt.permissionOrg)
```

Comment:
Name changes in this file were to disambiguate in the `authorizer_test` package.